docs/subnets/troubleshoot-deployments.md

```diff
@@ -0,0 +1,95 @@
+# Troubleshooting Subnet Deployments
+
+If you run into trouble deploying your Subnet, use this document for tips to resolve common issues.
+
+## Deployment Times Out
+
+During a local deployment, your network may fail to start. Your error may look something like this:
+
+```text
+[~]$ avalanche subnet deploy mySubnet
+✔ Local Network
+Deploying [mySubnet] to Local Network
+Backend controller started, pid: 26388, output at: /Users/user/.avalanche-cli/runs/server_20221231_111605/avalanche-cli-backend
+VMs ready.
+Starting network...
+..................................................................................
+..................................................................................
+......Error: failed to query network health: rpc error: code = DeadlineExceeded desc = context deadline exceeded
+```
+
+Avalanche-CLI only supports running one local Avalanche network at a time. If other instances of
+AvalancheGo are running concurrently, your Avalanche-CLI network fails to start.
+
+To test for this error, start by shutting down any Avalanche nodes started by Avalanche-CLI.
+
+```shell
+avalanche network clean --hard
+```
+
+Next, look for any lingering AvalancheGo processes with:
+
+```shell
+ps aux | grep avalanchego
+```
+
+If any processes are running, you need to stop them before you can launch your VM with Avalanche-CLI.
```

Comment:
eg killall avalanchego

Reply:
Very dangerous for things like line 40, if there are other avago running on the same machine

Reply:
I agree with Fabio, I think that's too dangerous. I think it's safe that developers will know how to kill a process.